src/main_functions_batch.c

```diff
@@ -69,27 +83,35 @@ char* edit_batch(char* line, dict_t* dict, int verbosity, int lnum)
 }
 
 /* See main_functions_batch.h */
-char** batch_mode(char* filename, dict_t* dict, bool* quit, int verbosity)
-{
-	if (verbosity == VERBOSE_MODE) printf("\n");
+char **batch_mode(char *filename, dict_t *dict, bool *quit, int verbosity) {
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
`*quit` is called here because the batch_mode should change its value, initially malloc'd in main.c, in the case that parse_file fails, no file is read and the function returns (this is the conditional checked in line 254). If I just called the value, I wouldn't be able to verify that batch_mode editing failed. batch_mode is one of a few functions that can change *quit value.
I've changed the function calls for functions who won't affect the value, such as the many you commented on for shellstrings, but I think for batch_mnode it might be necessary to do `*quit.